pontoon/base/views.py

```diff
@@ -708,59 +708,35 @@ def translation_memory(request):
     max_dist = math.floor(min(length / min_quality, 1000))
 
     # Only check entities with similar length
-    entities = Entity.objects.all().extra(
-        where=["CHAR_LENGTH(string) BETWEEN %s AND %s" % (min_dist, max_dist)])
+    entries = TranslationMemoryEntry.objects.filter(locale=locale).extra(
+        where=['(CHAR_LENGTH(source) BETWEEN %s AND %s)'\
+              'AND levenshtein_ratio(source, %s) > %s'],
+        params=(min_dist, max_dist, text, min_quality),
+        select={'quality': 'levenshtein_ratio(source, %s) * 100'},
+        select_params=(text,))
```

Comment:
Remove `,` after text and move final `)` to a new line (aligned with `entries`).